oem/firstvoices/common/build_keyboards.sh

```diff
@@ -1,11 +1,9 @@
 #!/bin/bash
 
-# Build the keyboards referred to in ../keyboards.csv and copy them + metadata 
+# Build the keyboards referred to in ../keyboards.csv and copy them + metadata
 # to the FirstVoices/Keyboards folder
-
-# This build script assumes that the https://github.com/keymanapp/keyboards repo is in
-# the same parent folder as this repo, with the default name 'keyboards'
-
```

Comment:
These lines should be edited not removed -- the script can still do the copy for now (we may remove that in future but I can see benefit for local testing)